masterlist.yaml

```diff
@@ -23380,7 +23380,8 @@ plugins:
       - 'Paradise_City_Rorikstead.esp'
       - name: 'The Great Cities - Minor Cities and Towns.esp'
         condition: 'checksum("JKs Skyrim.esp", 932DCD5F) or not active("The Great Cities of JK''s (North|Skyrim) - Patch.esp")'
-      - 'The Great City of Dawnstar.esp'
+      - name: 'The Great City of Dawnstar.esp'
+        condition: 'not active("COTN Dawnstar - JK''s Skyrim - TGC Dawnstar Patch.esp")'
```

Comment:
Can you please post a link to this patch?

Reply:
https://www.nexusmods.com/skyrimspecialedition/mods/30885?tab=files&file_id=143865
It's part of the Cities of the North - Dawnstar Patch Collection

Reply:
https://www.nexusmods.com/skyrimspecialedition/mods/30885?tab=files&file_id=444334
Sorry posted the wrong one, that's an old one I think with a different plugin name. This is the one automatically installed by the fomod